locale/en/common.po

```diff
@@ -370,6 +370,9 @@ msgstr "Delete Selection"
 msgid "common.deselect"
 msgstr "Deselect"
 
+msgid "common.deselectAll"
+msgstr "Deselect All"
```

Comment:
@jardakotesovec I'm not sure if your UI implementation uses this and the entry below. If not I'll remove them